CONTRIBUTING.md

```diff
@@ -45,6 +45,11 @@ What constitutes a business day is managed by the gem `business_time` and is con
 
 ### App setup
 
+Before running bin/setup, ensure that 'foreman' is removed from the Gemfile.  
+
+Prior to running 'bin/setup', it may be necessary to launch the postgres server using 
+```postgres -D /usr/local/var/postgres``` You can also use the OS X app.
```

Comment:
A link and some explicit instructions would be great, especially for newbies like us :)